src/Sylius/Bundle/ApiBundle/Resources/config/routing/product_option.yml

```diff
@@ -1,37 +1,13 @@
 # This file is part of the Sylius package.
 # (c) Paweł Jędrzejewski
 
-sylius_api_product_option_index:
-    path: /
-    methods: [GET]
-    defaults:
-        _controller: sylius.controller.product_option:indexAction
-        _sylius:
-            serialization_version: $version
-            paginate: $limit
-            sortable: true
-
-sylius_api_product_option_create:
-    path: /
-    methods: [POST]
-    defaults:
-        _controller: sylius.controller.product_option:createAction
-        _sylius:
-            serialization_version: $version
-
-sylius_api_product_option_update:
-    path: /{id}
-    methods: [PUT, PATCH]
-    defaults:
-        _controller: sylius.controller.product_option:updateAction
-        _sylius:
-            serialization_version: $version
-
-sylius_api_product_option_delete:
-    path: /{id}
-    methods: [DELETE]
-    defaults:
-        _controller: sylius.controller.product_option:deleteAction
-        _sylius:
-            serialization_version: $version
-            csrf_protection: false
+sylius_api_product_option:
+    resource: |
+        identifier: code
+        alias: sylius.product_option
+        section: api
+        only: ['index', 'show']
```

Comment:
While working on #7547, I realized that creating, updating and deleting product options through the API is no longer possible. Is this intentional or an oversight?